submissions/Olumide Ologbon/html/index.html

```diff
@@ -0,0 +1,172 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>
+            shopal
+        </title>        
+        <link href="style.css" rel="stylesheet">
+    </head>
+    <header class="head">
+        <div>
+            <div class="dropdown">
+                <button class="dropbtn">More</button>
+                <div class="dropdown-ctn">
+                    <a href="index.html">Home</a>
+                    <a href="#">About</a>
+                    <a href="#">Clients</a>
+                    <a href="#">Contact Us</a>
+                </div>
+            </div>   
+            <div class="dropdown">
+                <h1 style="margin-left: 30px; color: black;"> 
+                    Shopal Online<img src="image/shopal.png" alt="logo">
+                </h1>
+            </div>
+            <div class="dropdown">
+                <button class="dropbtn" style="margin-left: 930px ;">Account</button>
+                <div class="dropdown-ctn" style="margin-left: 930px ;">
+                    <a href="signup.html">Sign up </a>
+                    <a href="#">Login</a>
+                    <a href="#">Track Orders</a>
+                    <a href="#">Wish List</a>
+                </div>
+            </div>    
+        </div>        
+    </header>
+    <body>
+        <div>   
+            <img src="image/sales1.png" alt="promo!!!" class="img1">        
+        </div>
+
+        <div class="flex-container">
+            <div class="flex-item">
+                <h3>
+                    Featured categories
+                </h3>
+            </div>
+            <div class="flex-item">                
+                <img src="image/shoes.png" alt="shoes"><br>
+                Shoes                    
+            </div>
+            <div class="flex-item">
+                <img src="image/shirts.png" alt="shirt"><br>
+                Shirts
+            </div>
+            <div class="flex-item">
+                <img src="image/bags.png" alt="bags"><br>
+                Bags
+            </div>
+            <div class="flex-item">
+                <img src="image/electronics.png" alt="electronics"><br>
+                Electronics                    
+            </div>
+            <div class="flex-item">
+                <img src="image/phones.png" alt="Mobile phones"><br>
+                Mobile Phones
+            </div>
+            <div class="flex-item">
+                <img src="image/watch.png" alt="watch"><br>
+                Wrist Watches
+            </div>
+            <div class="flex-item">
+                <h3>
+                    Top Selling Items
+                </h3>
+            </div>
+            <div class="flex-item">
+                <img src="image/topsneakers.jpg" alt="topsneakers"><br>
+                Mens Fashion Sneakers  <br>&#8358 4,999                  
```

Comment:
Hey Olumide, the symbols usually end with a semicolon like this `&#8358;`. Did you notice how this affected the formatting/display of the mark ups beyond this line? Please update

Reply:
@kenny-wealth i have made the requested changes